src/codemagic/tools/_app_store_connect/action_group.py

```diff
@@ -22,3 +22,7 @@ class AppStoreConnectActionGroup(cli.ActionGroup):
         name='beta-build-localizations',
         description='Manage your beta builds localizations in App Store Connect',
     )
+    BETA_GROUPS = cli.ActionGroupProperties(
+        name='beta-groups',
+        description='Manage your groups of beta testers in App Store Connect',
```

Comment:
Maybe our wordsmith @helinanever can suggest something based on the overview given [here](https://developer.apple.com/documentation/appstoreconnectapi/prerelease_versions_and_beta_testers/beta_groups#overview). The description given here is used as the documentation for the corresponding tool action group in [web docs](https://github.com/codemagic-ci-cd/cli-tools/tree/master/docs/app-store-connect#action-groups) and in command line.

Reply:
Thi sounds good to me, let's keep what you have.